lib/pause_2017/PAUSE/Web/Context.pm

```diff
@@ -40,6 +41,17 @@ sub version {
   $version;
 }
 
+sub authenticator_for {
+  my ($self, $user) = @_;
+  my $cpan_alias = lc($user->{userid}) . '@cpan.org';
```

Comment:
I don't think we should add the `@cpan.org` bit. I don't use a cpan email address and this confuses me. I would just have it be the PAUSE login name

Reply:
Removed `@cpan.org` part with https://github.com/andk/pause/pull/455/commits/abb0141293fe982fd641110611e3da0fb04bca1a